crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -1560,7 +1617,7 @@ where
     ) -> EthResult<Option<Bytes>> {
         if let Some(block) = self.block_with_senders(block_id.into()).await? {
             if let Some(tx) = block.transactions().nth(index.into()) {
-                return Ok(Some(tx.envelope_encoded()))
+                return Ok(Some(tx.envelope_encoded()));
```

Comment:
absolutely losing it with these semicolons

Reply:
legit crazy how this is still a heisenbug in the formatter lol